bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_rooworkspace.py

```diff
@@ -33,10 +33,33 @@
 */
 """
 
+from ._utils import _kwargs_to_roocmdargs
+
 
 class RooWorkspace(object):
+    def __init__(self, *args, **kwargs):
+        # Redefinition of `RooWorkspace` constructor for keyword arguments.
+        # The keywords must correspond to the CmdArg of the constructor function.
+        args, kwargs = _kwargs_to_roocmdargs(*args, **kwargs)
+        self._init(*args, **kwargs)
+
+    def __getitem__(self, key):
+        # To enable accessing objects in the RooWorkspace with dictionary-like syntax.
+        # The key is passed to the general `RooWorkspace::obj()` function.
+        return self.obj(key)
+
     def Import(self, *args, **kwargs):
         """
         Support the C++ `import()` as `Import()` in python
         """
         return getattr(self, "import")(*args, **kwargs)
+
+
+def RooWorkspace_import(self, *args, **kwargs):
+    # Redefinition of `RooWorkspace.import()` for keyword arguments.
+    # The keywords must correspond to the CmdArg of the `import()` function.
+    args, kwargs = _kwargs_to_roocmdargs(*args, **kwargs)
```

Comment:
No big deal but why not just adding this line to the implementation of `Import` above? Why the need of an extra injection of another function? Is it because you also expect the user to call `import` from Python and not `Import`?

Reply:
@etejedor Yes Sir, since RooWorkspace.Import is a wrapper function generated by cppyy to avoid having a function with the keyword name import, we have't directly defined a function called import in the RooWorkspace mirror class because import is a Python keyword